### ISSUE TITLE
Conflict with new SdFat library

### ISSUE THREAD

**Issue by oldmanegan** [bneedhamia/sdconfigfile#5]
Have you tried your library with this:
https://github.com/greiman/SdFat

I encountered several issues and eventually removed your library after attempting to fix and will have to write my own config type code apparently and unfortunately.

Also, I suggest posting a means (in a sketch) of saving to a config file so the utility of the work is greater - a user could create the file on the fly in a sketch and after a reboot get the changed variables back... somewhat different than your original application, but needed for those with an SD card that wish to avoid PROGMEM or other means.  Thanks

**Comment by bneedhamia** [comment]
I just now tried replacing the SD library with SdFat, and unfortunately found that SdFat is not code-compatible with SD. That is, I don't know of any code that can easily support linking with either the SD library or the SdFat library based on the user's Arduino Sketch.

If you still want to pursue using SdFat in sdconfigfile, here are the set of changes I noticed are required: 1) in the example .ino file, replace #include <SD.h> with #include <SdFat.h>; 2) do the same #include replacement in SDConfigFileExample.ino; 3) Replace occurrences of "SD." with "SdFileSystem."; 4)) add the necessary parameter to the call to SdFileSystem.begin(). I didn't pursue the changes necessary beyond that - sorry.
